src/components/UnlockedWalletLayout.tsx

```diff
@@ -58,88 +63,105 @@ const WalletLayout: FC = ({ children }) => {
   const activeWalletNameInitials = getInitials(activeWalletName)
 
   return (
-    <WalletContainer initial={{ opacity: 0 }} animate={{ opacity: 1 }} transition={{ duration: 0.8 }}>
-      <AppHeader>
-        {networkStatus === 'online' && (
-          <RefreshButton
+    <>
+      <motion.div
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        transition={{ duration: 0.8 }}
+        className={className}
+      >
+        <WalletSidebar>
+          <CurrentWalletInitials
+            onClick={() => setIsNotificationsModalOpen(true)}
+            style={{ zIndex: isNotificationsModalOpen ? 2 : undefined }}
+          >
+            {activeWalletNameInitials}
+          </CurrentWalletInitials>
+          <SideNavigation>
+            <NavItem Icon={Layers} label={t`Overview`} to="/wallet/overview" />
+            <NavItem Icon={ArrowLeftRight} label={t`Send`} onClick={() => openSendModal(TxType.TRANSFER)} />
```

Comment:
Just a suggestion, not related to a bug: in the navbar it would be great to make the data-tip to appear on the right, sticked to the parent element (anchor on the right hand side) :)